### ISSUE TITLE
mapInstanceState doesn't support simple object map

### ISSUE THREAD

**Issue by jcupps** [brophdawg11/vuex-helpers#13]
Hi, thanks for the library. I ran into a bug in which the `mapInstanceState` helper throws an error when using a simple key/value object map.

As shown in the [Vuex docs](https://vuex.vuejs.org/guide/state.html#the-mapstate-helper), the following should be supported:
```javascript
mapInstanceState(vm => vm.someNamespace, {
    // passing the string value 'count' is same as `state => state.count`
    countAlias: 'count'
})
```

Error:
> TypeError: mapper.call is not a function
Thrown here: https://github.com/brophdawg11/vuex-helpers/blob/master/src/vuex-helpers.js#L39

It looks like this could be fixed by changing that line to the following:
```javascript
return typeof mapper === 'function' ? mapper.call(this, moduleState) : moduleState[mapper];
```

Let me know if you'd like me to open a PR.

**Comment by brophdawg11** [comment]
Hey!  Yeah that's sort of "by design" since we only ever use the function signature - but would be happy to accept a PR!  I actually just left URBN last week for a new role - so I'll reach out to one my former coworkers since I'll need one of them to publish a package once we merge the PR.  There's also a good chance we'll fork this over to a public repo in the URBN organization - I would have done that before I left but wasn't aware that anyone was using this - so glad to hear it's been useful outside of URBN!

**Comment by brophdawg11** [comment]
Out of curiosity - what's your use case for this?  I always thought it was a super useful concept and was a bit surprised when it didn't gain traction for the core Vuex library, so am interested in other use cases where this proved helpful.

**Comment by brophdawg11** [comment]
Thanks for the PR - I'll see if I can get reach out and get this published shortly 👍

**Comment by brophdawg11** [comment]
Published 0.2.0: https://www.npmjs.com/package/@urbn/vuex-helpers
